### DISCUSSION TITLE
testing time facing issue

### DISCUSSION THREAD

**Discussion by appasaheb4** [mobxjs/mobx#3426] [General]
App development time working properly .
But testing time failed test case 
please check ss
<img width="620" alt="Screenshot 2022-06-13 at 2 02 02 PM" src="https://user-images.githubusercontent.com/24726539/173313150-58bf456e-d9f2-4799-b011-17094e6d81fa.png">

**Comment by iChenLei** [comment]
Sorry, we can't help you to debug your issue . Information is not enough. What is `useStores()` ? Like below code ?
```js
import { useContext } from "react";
import { MobxProviderContext } from "mobx-react";

export function useStores () {
    return useContext(MobxProviderContext);
}
```

**Comment by kubk** [comment]
Hi @appasaheb4 
Please try to reproduce the issue here: https://codesandbox.io/s/minimal-mobx-react-project-ppgml?file=/index.js
